playbooks/os_updates.yml

```diff
@@ -0,0 +1,29 @@
+---
+- hosts: all
```

Comment:
Update this to run by default on the `staging` group, with an option to run on `qa` or `production` if you pass a variable at runtime.